tokens/global/switch.json5

```diff
@@ -0,0 +1,224 @@
+{
+  color: {
+    docCategory: 'colors',
+    docExample: 'color',
+    background: {
+      switch: {
```

Comment:
I wonder if we are missing the variant here

'color-background-switch-primary-rest'
would allow us to add additional variants in the future without impacting these tokens

Reply:
Default makes sense, especially if it's used on chip. I'll go ahead and add that